getting_started/step_by_step/signals.rst

```diff
@@ -326,7 +326,7 @@ bottom of our script and use it to toggle our sprite's visibility.
 
  .. code-tab:: csharp C#
 
-    private void OnTimerTimeout()
+    private void _on_timer_timeout()
```

Comment:
Same for this, see on the example above

Reply:
Okay I was very dumb. Please close this out. Following the tutorial I didn't look at the obvious of editing the reciever method.